lp-core-platform/lp-cp-xwiki-implementation/src/main/java/eu/learnpad/core/impl/mv/XwikiController.java

```diff
@@ -102,6 +104,17 @@ public synchronized void updateBridgeInterface (BridgeInterface bi){
 	@Override
 	public void notifyVerification(String verificationProcessId)
 			throws LpRestException {
+
+	    VerificationStatus status = this.bridge.getVerificationStatus(verificationProcessId);
+	    //TODO: show the status.getStatus() somewhere in the wiki for the verification with id verificationProcessId?
+	    
+	    if(!status.getStatus().equals(StatusType.COMPLETED)){
```

Comment:
Our hypothesis was: if the verification process fail, then the model is not imported. But indeed, we didn't discuss who to notify about a failure. Let's keep this comment for now and take care of that later (unless @gulyx has some idea?).